packages/firebase_ml_vision/test/firebase_ml_vision_test.dart

```diff
@@ -148,6 +151,10 @@ void main() {
         // TODO(jackson): Use const Rect when available in minimum Flutter SDK
         // ignore: prefer_const_constructors
         expect(barcode.boundingBox, Rect.fromLTWH(1.0, 2.0, 3.0, 4.0));
+        expect(
+          barcode.rawBytes,
+          Uint8List.fromList([104, 101, 108, 108, 58, 114, 97, 119]),
```

Comment:
It'd be great to have an e2e test for this, to test the native side too.

Reply:
@rrousselGit I added a test to `packages/firebase_ml_vision/example/test_driver/barcode_detector.dart` - let me know if this will suffice? :)